changelog/changelog.ddoc

```diff
@@ -8,8 +8,7 @@ CHANGELOG_NAV_FIRST=$(DIVC changelog-nav, next version: $(A $1.html, $1))
 CHANGELOG_NAV_LAST=$(DIVC changelog-nav, previous version: $(A $1.html, $1))
 
 SUBNAV=$(SUBNAV_TEMPLATE
-    $(SUBNAV_HEAD Change Log, $(ROOT_DIR)changelog/index.html, List of all versions,
-        subnav-duplicate)
+    $(SUBNAV_HEAD Change Log, $(ROOT_DIR)changelog/index.html, List of all versions)
```

Comment:
The correct fix is add a forth empty parameter to the `SUBNAV_HEAD` macro invocation, though I'm not well-enough versed with ddoc to know if simply:
```
$(SUBNAV_HEAD Change Log, $(ROOT_DIR)changelog/index.html, List of all versions, )
```
would work.